types/cookie-session/index.d.ts

```diff
@@ -84,17 +84,17 @@ declare namespace CookieSessionInterfaces {
         /**
          * Is true if the session has been changed during the request.
          */
-        isChanged?: boolean;
+        isChanged: boolean;
```

Comment:
These three named properties [isChanged, isNew, isPopulated] should remain optional. This will not break current code, and remain in the spirit of the cookie-session examples.